VERSION.cmake

```diff
@@ -1,4 +1,4 @@
-set (DEFAULT_DNF_VERSION "4.2.23")
+set (DEFAULT_DNF_VERSION "4.2.24")
```

Comment:
Hmm, why do you bump the dnf version? It probably doesn't hurt but it's not required for anything?

Reply:
Yeah, I don't think it's needed, so I removed it.